spec/datadog/core/configuration/settings_spec.rb

```diff
@@ -12,6 +12,10 @@
 RSpec.describe Datadog::Core::Configuration::Settings do
   subject(:settings) { described_class.new(options) }
 
+  after do
+    settings.reset!
+  end
```

Comment:
Similar to my comment above on ci/configuration/settings_spec.rb, I don't think this is needed? And the specs still pass for me if I remove this.